backend/backend.go

```diff
@@ -27,6 +29,22 @@ type Backend interface {
 	State() (state.State, error)
 }
 
+// MultiState is an interface that a backend can implement to allow changing
+// between named states depending on the configured environment.
+type MultiState interface {
+	// States returns a list of configured named states and the current state.
+	States() ([]string, string, error)
```

Comment:
Ah I can see how that is confusing. Sorry!

Given that API, maybe the behavior should be to remove `ChangeState` and just have `State(name)`, `DeleteState(name)`, `States() []string error`?